### ISSUE TITLE
filter out --drb warning

### ISSUE THREAD

**Issue by chalmagean** [duskhacker/sweet-rspec-vim#5]
If you're using --drb in you .rspec file, vim opens the fix window and shows you the warning which is pretty annoying.

Should probably be filtered out somehow.

Cezar

**Comment by ivanoats** [comment]
Yeah, if all test pass, this makes the plugin not seem to work at all. All I get is the no -drb warning
